components/formatter/RancherClusterName.vue

```diff
@@ -0,0 +1,53 @@
+<script>
+import LinkDetail from '@/components/formatter/LinkDetail';
+
+export default {
+  components: { LinkDetail },
+  props:      {
+    value: {
+      type:     String,
+      default: ''
+    },
+    row: {
+      type:     Object,
+      required: true
+    },
+    reference: {
+      type:    String,
+      default: null,
+    }
+  },
+  computed: {
+    inactiveNodes() {
```

Comment:
Nodes (`MANAGEMENT.NODE`) are for all non-RKE2 clusters. For RKE2 you'll need to check the state of a cluster's`CAPI.MACHINE` (otherwise user would may get confused when clicking on the cluster and seeing different in the machine/machine pool list). There's an example of how to filter machines down for those in a specific cluster in the computed property `machines` in `detail/provisioning.cattle.io.cluster.vue`. If you're feeling brave that and the `nodes` one underneath could use the underlying cluster model ones (would still need to add the fake nodes/machines in the computed properties though)

Reply:
Actually, just woken up. There's a wider issue here, we show machine state for rke2 clusters in the machine/node column. We didn't do that before in Ember, just showed the count for RKE1 nodes. Think we should either...

1) Always show the icon for machine / node warning in the cluster node column as per ember
2) Move the warning to the Machine/Node column and not show for RKE2 clusters
3) Not show the warning at all and have a similar status bar for RKE1 nodes as RKE2 machines

Given the time in the release, I'd probably go for option number one. @lvuch Would this be ucky design wise?